test/metabase/api/card_test.clj

```diff
@@ -2943,10 +2944,10 @@
                                            uploads-database-id (mt/id)
                                            uploads-table-prefix nil
                                            uploads-schema-name "PUBLIC"]          (let [{:keys [status body]} (upload-example-csv-via-api!)]
-            (is (= 200
-                   status))
-            (is (= body
-                   (t2/select-one-pk :model/Card :database_id (mt/id)))))))
+                                                                                   (is (= 200
```

Comment:
this let belongs on the newline, I think.

Reply:
oops, how'd that happen